Assets/Scripts/Battlefield/Camera/CameraTarget.cs

```diff
@@ -19,7 +19,11 @@ void Start()
     
         void Update()
         {
-            transform.position = player.position;
+            if(player != null)
```

Comment:
I think this camera should be destroyed as well when the player dies so there arent extra game object. Could you implement a function or apply an abstract class that units or creatures derive from that have an OnDeath() or something to kill destroy this camera along with itself.

Reply:
I didn't write these classes so I'm not entirely sure how Jack has them set up. I just spotted the bug and made a quick fix, so Jack will probably have to do that

Reply:
It's only 3 lines of code so we should just merge it in so we aren't getting null reference exceptions for timebox 2 then worry about destroying the cameras for timebox 3

Reply:
It's also still best practice IMO to check if an object exists before trying to reference it anyway, so I think we should leave this check and then add the code for destroying cameras later on

Reply:
Sorry, i didnt see this earlier. Yes, I think it should be ok for now. I will accept.